databricks/koalas/tests/test_dataframe.py

```diff
@@ -88,6 +88,68 @@ def test_dataframe(self):
         index_cols = pdf.columns[column_mask]
         self.assert_eq(kdf[index_cols], pdf[index_cols])
 
+    def test_insert(self):
+        #
+        # Basic DataFrame
+        #
+        pdf = pd.DataFrame([1, 2, 3])
+        kdf = ks.from_pandas(pdf)
+
+        kdf.insert(1, "b", 10)
+        pdf.insert(1, "b", 10)
+        self.assert_eq(kdf.sort_index(), pdf.sort_index(), almost=True)
+
+        kdf.b = kdf.b + 1
+        pdf.b = pdf.b + 1
+        self.assert_eq(kdf.sort_index(), pdf.sort_index(), almost=True)
```

Comment:
Here `kdf` doesn't recognize `kdf.b`'s change.

Reply:
I'm not sure what you mean? Seems like the tests passed.

Reply:
Btw, I'm also not sure what you want to test here.

Reply:
In my local,
```
Left: # koalas
   0   b
0  1  10
1  2  10
2  3  10
0    int64
b    int32
dtype: object

Right: # pandas
   0   b
0  1  11
1  2  11
2  3  11
0    int64
b    int64
dtype: object
```
After column b is inserted to kdf, `kdf.b = kdf.b + 1` can't be recognized by `kdf`, so column `b` remains 10 rather than 11 in the example.

Reply:
It surprised me CI jobs pass now.

Reply:
```
>>> kdf = ks.DataFrame({'x': [1, 2, 3]})
>>> kdf
   x
0  1
1  2
2  3
>>> kdf.x = kdf.x + 1
>>> kdf = ks.DataFrame({'x': [1, 2, 3]})
>>> kdf.x = kdf.x + 1
>>> kdf.x
0    2
1    3
2    4
Name: x, dtype: int64
>>> kdf # wondering why
   x
0  1
1  2
2  3
>>> pdf = pd.DataFrame({'x': [1, 2, 3]})
>>> pdf.x = pdf.x + 1
>>> pdf.x
0    2
1    3
2    4
Name: x, dtype: int64
>>> pdf 
   x
0  2
1  3
2  4
```

Reply:
I guess you should merge master? Seems like this branch doesn't include the commit by #1989.

Reply:
Got it, thanks!